src/Diagnostics.DataProviders/DataProviders/SupportObserverDataProvider.cs

```diff
@@ -242,18 +241,32 @@ public override async Task<DataTable> ExecuteSqlQueryAsync(string cloudServiceNa
             };
 
             var response = await SendObserverRequestAsync(request);
-            response.EnsureSuccessStatusCode();
+
+            try
+            {
+                response.EnsureSuccessStatusCode();
+            }
+            catch (HttpRequestException ex)
+            {
+                Logger.LogDataProviderMessage(RequestId, "ObserverDataProvider", $"message:Observer SQL query request failed, query:{query}, error:{ex.Message}");
```

Comment:
My only recommendation here is to log instead the response code and the response body in the case the body has content. Since your throwing the exception, the DataLogProviderDecorator will already log that exception